CONTRIBUTING.md

```diff
@@ -104,21 +104,17 @@ https://stestr.readthedocs.io/en/stable/MANUAL.html#test-selection
 If you want to run a single test module, test class, or individual test method you can
 do this faster with the `-n`/`--no-discover` option. For example, to run a module:
 ```
-tox -- -n test.python.test_examples
```

Comment:
These commands were not working since some environments such as lint did not recognize the option, so an environment has to be specified.